streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/resource/Resources.java

```diff
@@ -0,0 +1,132 @@
+package org.ray.streaming.runtime.core.resource;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.ray.api.id.UniqueId;
+import org.ray.streaming.runtime.config.master.ResourceConfig;
+
+/**
+ * Resource description of ResourceManager.
+ */
+public class Resources implements Serializable {
```

Comment:
Can we use a better name? Such as `ResourceSpec` or `ResourceDesc`

Reply:
There is no better name yet.